src/main/kotlin/org/rust/ide/typing/RsMetaTagParenthesisHandler.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.typing
+
+import com.intellij.codeInsight.CodeInsightSettings
+import com.intellij.codeInsight.editorActions.TypedHandlerDelegate
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.EditorModificationUtil
+import com.intellij.openapi.editor.ex.EditorEx
+import com.intellij.openapi.editor.highlighter.HighlighterIterator
+import com.intellij.openapi.fileTypes.FileType
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsElementTypes.*
+import org.rust.lang.core.psi.RsFile
+
+class RsMetaTagParenthesisHandler : TypedHandlerDelegate() {
```

Comment:
Let's rename it to `RsMetaItemParenthesisTypedHandler` because [we](https://github.com/intellij-rust/intellij-rust/blob/master/src/main/grammars/RustParser.bnf#L129-L135) and [rust](https://doc.rust-lang.org/reference/attributes.html) use `MetaItem` as a name of ast nodes inside attributes. Or even to `RsParenthesisTypedHandler` to avoid renaming when we need to add parenthesis completion in some other place in future